grails-app/services/org/pih/warehouse/dashboard/IndicatorDataService.groovy

```diff
@@ -1188,7 +1188,7 @@ class IndicatorDataService {
         String query = '''
             SELECT 
                 product_code,
-                COUNT(product_code) as products,
+                COUNT(DISTINCT backdated_shipment) as shipments_with_backdated_product,
```

Comment:
won't changing `products` to `shipments_with_backdated_product` affect whoever is using the result of this query? Where is this query getting parsed?

Reply:
Looks like line 1267 is where this is accessed and it's by index so the rename is probably fine. Please retest though just in case.

Reply:
works well